io/disk/rawread.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2019 IBM.
+# Author: Naresh Bannoth <nbannoth@in.ibm.com>
+
+"""
+Rawread test
+"""
+
+import os
+from avocado import Test
+from avocado import main
+from avocado.utils import archive
+from avocado.utils import build
+from avocado.utils import process, distro
+from avocado.utils.software_manager import SoftwareManager
+
+
+class Rawread(Test):
+
+    """
+    Rawread is a benchmark suite that is aimed at performing a number
+    of simple tests of hard drive like write and read
+    """
+
+    def setUp(self):
+        """
+        checking install of required packages and extract and
+        compile of rawread suit.
+        """
+        smm = SoftwareManager()
+        deps = ['gcc', 'make', 'libaio-devel']
+        if distro.detect().name == 'Ubuntu':
+            deps.extend(['g++'])
+        else:
+            deps.extend(['gcc-c++'])
+
+        for package in deps:
+            if not smm.check_installed(package) and not smm.install(package):
+                self.cancel("Fail to install Package: %s" % package)
+
+        tarball_path = os.path.join(os.path.abspath(''),
+                                    "rawread.py.data/rawread.tar")
```

Comment:
This is not needed. Please go through other examples.

Reply:
I have went through the couple of other test scripts, in those cases we have the url for the tool file, here we use the file from the data directory.
are you saying like below, 

locations = ["https://github.com/mkuoppal/tiobench/archive/master.zip"]
        tarball = self.fetch_asset("tiobench.zip", locations=locations)

I tried but it was erroring out, hence used the above thing.
can you suggest any specific file where can I check.

Reply:
@Naresh-ibm 
You have looked at the wrong examples. You are confusing downloading file with local file.
Did you look at examples that use `self.get_data()` ?

Reply:
got it. will change.
Thanks for it.